soc/xtensa/intel_adsp/common/include/cavs-link.ld

```diff
@@ -274,6 +274,11 @@ SECTIONS {
     *(.rodata.*)
     *(.gnu.linkonce.r.*)
     *(.rodata1)
+
+    . = ALIGN(4);
+    #include <snippets-rodata.ld>
+    . = ALIGN(4);
```

Comment:
Are both alignments required?  If they are actually required, I prefer you remove them (especially the 2nd one since there was no alignment previously for the __XT_EXCEPTION_TABLE__

Reply:
Arguably even if they are required, they belong in the snippet definitions anyway.  Zephyr linkage management gets a little wonky in places.  But yeah, it's hard to imagine that a 1-word alignment is being defeated somehow.

The core change here is for sure needed though; the cAVS linker script went through a big rework recently and clearly I missed this bit.

Reply:
Done